packages/dx-react-scheduler/docs/guides/fundamentals.md

```diff
@@ -0,0 +1,43 @@
+# React Scheduler - Fundamentals
+
+The [DevExtreme React Scheduler](https://devexpress.github.io/devextreme-reactive/react/scheduler) component that displays appointments and provide a functionality to manage data.
```

Comment:
DevExtreme React Scheduler **is a** component that displays...
or
The DevExtreme React Scheduler component displays...